lib/snapshot/snapshot_fuchsia.dart

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
```

Comment:
@rmacnak-google I can't find where this file and snapshot.dart in the same directory are used. Are they remnants of Dart 1?

Reply:
Right, this is leftover from Dart 1 and is replaced by libaries.json/yaml.